projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -107,7 +107,7 @@ control_service_integration_test:
 control_service_test_helm_chart:
   stage: build
   script:
-    - export DESIRED_VERSION=v3.6.3 # helm version 3.6.3
+    - export DESIRED_VERSION=v3.11.0 # helm version 3.11.0
```

Comment:
seems the PR merges multiple changes
(however cannot tell, since there is no exact PR description of what had changed exactly, and why)

Reply:
haven't we split that in two already in PR https://github.com/vmware/versatile-data-kit/pull/1547? why not merging it first in case a prerequisite?

Reply:
I am going to delete that PR. 
I never finished that PR out because it was hard to figure out how to effectively test it.